rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -783,7 +783,7 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		const QVariantList var_list = ui->audioOutBox->itemData(index).toList();
 		ensure(var_list.size() == 2 && var_list[0].canConvert<QString>());
 		const QString text = var_list[0].toString();
-		const bool enabled = text == "XAudio2" || text == "OpenAL" || text == "FAudio";
+		const bool enabled = text != "Null";
```

Comment:
I'd rather have this explicit.
It's better to add backends if necessary than having to remove them because they don't work.

Reply:
Reverted